content/docs/intro/console/extensions/ci-cd-integration-assistant.md

```diff
@@ -0,0 +1,124 @@
+**Configure VCS** step are complete click the **Next** button. The wizard will confirm
+that your VCS configuration has been recognized and will automatically move you to the
+next step.
+
+### Configure CI/CD secrets
+
+Now that your Pulumi project is configured to use a VCS your team can collaborate with you easily.
+Most importantly, your Pulumi project is safe from accidents on your local machine!
+
+If you start the CI/CD integration wizard having configured the VCS for your project already,
+the wizard will skip to the next step automatically. You can see the repository that your project
+is integrated with by clicking on the Configure VCS step marker.
+
+The **Configure CI/CD** step will help you configure secrets that will be used by your Pulumi stack.
+For supported services, the wizard will provide a convenient link to the respective location
+where you can configure the secrets.
+
+The wizard provides a convenient way to create a [Pulumi Access Token]({{< relref "/docs/intro/console/accounts-and-organizations/accounts#access-tokens" >}})
+without needing to leave the page.
+
+![Pulumi Access Token](/images/docs/reference/console/pulumi-access-token.png)
+
+In the following example the wizard is being used to configure a GitHub Actions workflow.
+So the wizard provides a direct link to configure secrets for your workflow.
+
+> See the [Cloud Providers]({{< relref "/docs/intro/cloud-providers" >}}) page to find the setup page for your cloud provider.
+
+### Add the workflow
+
+Once the secrets are configured, the next step gives you the relevant workflow to add to your repository.
+The workflow is customized for the current stack, so you can be sure that you are configuring a workflow that uses
+the correct stack configuration.
+
+![Add Workflow](/images/docs/reference/console/add-workflow.png)
+
+> Be sure to commit the new workflow file to a separate branch and not directly to your primary branch. This will create a new pull request for you.
+
+The workflow configuration provided by the wizard is configured to run a `pulumi preview` for pull request builds.
+Pull request builds help you catch problems _before_ the changes are merged -- a very important consideration for infrastructure
+that is likely hosting services critical to your business.
+
+### Validation
+
+In the previous step, you committed a new workflow configuration by creating a new pull request. This will trigger a new build
+that will run the `pulumi preview` command. Click the **Next** button to validate the CI configuration.
+
+Congratulations on configuring a version control system and an automated pipeline for your stack! 🎉
+
+> If your pull request build failed, use the [CI/CD troubleshooting guide]({{< relref "/docs/guides/continuous-delivery/troubleshooting-guide" >}}) to diagnose the issue.
+
+## Skip the line and get the workflow directly
+
+If you want to access the workflow for a specific CI/CD service and configure VCS on your own,
```

Comment:
Seems like we should link to our general CI/CD guide _somewhere_ here. Though it isn't clear where it would fit naturally.

```
<a href="{{< relref "/docs/guides/continuous-delivery" >}}">Continuous Delivery</a>
```

Reply:
I'll actually skip that here since the Console does have links to those guides already.